torchrl/envs/transforms/transforms.py

```diff
@@ -4083,8 +4082,8 @@ def __init__(
         """Initialises the transform. Filters out non-reward input keys and defines output keys."""
         if in_keys is None:
             in_keys = ["reward"]
-        if out_keys is None and in_keys == ["reward"]:
-            out_keys = ["episode_reward"]
+        if out_keys is None:
+            out_keys = [_replace_last(in_key, "episode_reward") for in_key in in_keys]
```

Comment:
what if the reward is named "serendipity"?

Reply:
by default the output will be named "episode_reward",
would you prefer "episode_serendipity"?